candidates/election_specific.py

```diff
@@ -10,6 +11,22 @@ def default_fetch_area_ids(**kwargs):
 
     return areas
 
+
+def default_fetch_posts_for_area(**kwargs):
```

Comment:
I may be missing something obvious here, but why did you need to move this code into `election_specific` - wouldn't you still get the same functionality and DRY advantage by factoring out `fetch_posts_for_area` into a function in `candidates/views/api.py`? (The implementation seems to be exactly the same in the UK-specific version as the default one, apart from the implementation of `fetch_area_ids` used, and that'd be dealt with by importing it from `election_specific`.)